pyo3-macros-backend/src/method.rs

```diff
@@ -21,6 +20,44 @@ pub struct FnArg<'a> {
     pub attrs: PyFunctionArgAttrs,
 }
 
+impl<'a> FnArg<'a> {
+    /// Transforms a rust fn arg parsed with syn into a method::FnArg
+    pub fn parse(arg: &'a mut syn::FnArg) -> syn::Result<Self> {
```

Comment:
A bit nitpicking, but syn has a [Parse::parse](https://docs.rs/syn/latest/syn/parse/trait.Parse.html) which takes `TokenStream` and returns `Option<Self>`.
So, isn't this naming confusing?
I think the boring `from_*` style naming is better (e.g.,  `from_syn_fnarg`).